COMMERCE-11938

victorcdp opened the pull request:
https://liferay.atlassian.net/browse/COMMERCE-11938

noornajj commented:
ci:test:sf

liferay-continuous-integration commented:
You cannot perform that action because you are not a member of the Liferay organization. Please make sure that you have been added and that your organization membership is set as Public. See https://help.github.com/articles/publicizing-or-hiding-organization-membership for more information.

noornajj commented:
ci:test:relevant

liferay-continuous-integration commented:
You cannot perform that action because you are not a member of the Liferay organization. Please make sure that you have been added and that your organization membership is set as Public. See https://help.github.com/articles/publicizing-or-hiding-organization-membership for more information.

noornajj commented:
ci:test:sf

brianikim commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-9.liferay.com/job/test-portal-source-format/1674/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-9.liferay.com//userContent/jobs/test-portal-source-format/builds/1674/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7176/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7176/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8eebd3a6312a6b899d0feac6176a90ba479ce866">8eebd3a6312a6b899d0feac6176a90ba479ce866</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/noornajj/liferay-portal/tree/LPS-115642">LPS-115642</a><br/>Branch GIT ID: <a href="https://github.com/noornajj/liferay-portal/commit/d0b9c607f9464633125135a0d246a2995c03047e">d0b9c607f9464633125135a0d246a2995c03047e</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-9.liferay.com/job/test-portal-source-format/1674/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-9.liferay.com/userContent/jobs/test-portal-source-format/builds/1674/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 23 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8eebd3a6312a6b899d0feac6176a90ba479ce866">8eebd3a6312a6b899d0feac6176a90ba479ce866</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/132b6a6de2529323e7ade179ae4e1386e97be5cb">132b6a6de2529323e7ade179ae4e1386e97be5cb</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7176/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7176/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7176/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':baseline3'.
     [exec] &gt; Could not resolve all files for configuration ':detachedConfiguration1'.
     [exec]    &gt; Could not resolve com.liferay.portal:com.liferay.portal.kernel:[3.0.0,4.0.0).
     [exec]      Required by:
     [exec]          project :
     [exec]       &gt; Failed to list versions for com.liferay.portal:com.liferay.portal.kernel.
     [exec]          &gt; Unable to load Maven meta-data from https://repository-cdn.liferay.com/nexus/content/groups/public/com/liferay/portal/com.liferay.portal.kernel/maven-metadata.xml.
     [exec]             &gt; Could not get resource 'https://repository-cdn.liferay.com/nexus/content/groups/public/com/liferay/portal/com.liferay.portal.kernel/maven-metadata.xml'.
     [exec]                &gt; Could not GET 'https://repository-cdn.liferay.com/nexus/content/groups/public/com/liferay/portal/com.liferay.portal.kernel/maven-metadata.xml'.
     [exec]                   &gt; Read timed out
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':baseline3'.
     [exec] 	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:38)
     [exec] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter\$1.executeTask(EventFiringTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter\$1.call(EventFiringTaskExecuter.java:55)
     [exec] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter\$1.call(EventFiringTaskExecuter.java:52)
     [exec] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor\$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:416)
     [exec] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor\$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:406)
     [exec] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor\$1.execute(DefaultBuildOperationExecutor.java:165)
     [exec] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:250)
     [exec] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:158)</code></pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/7176/">test-portal-acceptance-pullrequest(master)#7176</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-11.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7176/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/99">brianikim#99</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/896.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/11835.aspx">master - noornajj > brianikim - PR#99</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-11.liferay.com/job/publish-spira-report/2325/">publish-spira-report#2325</a><br />

brianikim commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6489/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-14.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6489/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 20 out of 20 jobs passed</h3><h3>:heavy_check_mark: ci:test:relevant - 45 out of 45 jobs passed in 1 hour 40 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3efd4a911e7f1634d2bb7ef41328ecb45c9b4139">3efd4a911e7f1634d2bb7ef41328ecb45c9b4139</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/132b6a6de2529323e7ade179ae4e1386e97be5cb">132b6a6de2529323e7ade179ae4e1386e97be5cb</a></p><details><summary><strong>ci:test:stable - 20 out of 20 jobs PASSED</strong></summary><details><summary><strong>20 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160962/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/372702/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430329/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160963/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402819/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/371649/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402821/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/296145/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160967/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400388/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/330093/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430330/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402822/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400635/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406273/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407633/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/296146/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160965/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400387/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407634/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><details><summary><strong>ci:test:relevant - 45 out of 45 jobs PASSED</strong></summary><details><summary><strong>45 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6489/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160964/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406272/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160962/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/278887/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400386/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402820/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400636/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/350553/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/372701/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/278886/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406271/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/371648/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430328/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406270/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407631/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400385/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/372702/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430329/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/1</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160963/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/10</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402819/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/11</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/371649/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/2</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402821/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/3</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/296145/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/4</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160967/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/5</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400388/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/330093/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/430330/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/8</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/402822/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8_stable/9</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/372700/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/296144/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407629/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/371650/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/382065/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400635/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8_stable</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407630/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/330092/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/406273/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8_stable</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407633/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/296146/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160965/">test-portal-acceptance-pullrequest-batch(master)/rest-builder-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407632/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400387/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/160966/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/407634/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8_stable</a></li></ul></details></details><h5>For more details click <a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6489/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest(master)/6489/">test-portal-acceptance-pullrequest(master)#6489</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-14.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6489/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> relevant<br /><strong>Pull Request:</strong><a href="https://github.com/brianikim/liferay-portal/pull/99">brianikim#99</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/12/Release/896.aspx">/Liferay DXP 7.3/Pull Request/ci:test:relevant</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/12/Build/11835.aspx">master - noornajj > brianikim - PR#99</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-11.liferay.com/job/publish-spira-report/2419/">publish-spira-report#2419</a><br />

brianikim commented:
https://github.com/SamZiemer/liferay-portal/pull/742

kyle-miho committed:
LRQA-69418 Add automation to cover deploying Service Builder modules with JDK11

andrea-ale-sbarra committed:
COMMERCE-11702 forcePredictableOperationId true (yaml file)

andrea-ale-sbarra committed:
COMMERCE-11702 autogenerated code (gw buildREST)

andrea-ale-sbarra committed:
COMMERCE-11072 fixed methods signature

victorhcunha committed:
LPS-186452 Add CanExportObjectEntries test

Commit:
LPS-186154 add test

georgel-pop-lr committed:
LPS-188294 Fix duplicated web contents when filtering with Recent and Status

victorhcunha committed:
LPS-186452 Add CanDeleteObjectEntries test

victorhcunha committed:
LPS-186468 Add CanAnonymizeObjectEntries test

victorhcunha committed:
LPS-187082 Add CanDeleteOriginalFieldFromEntry test

victorhcunha committed:
LPS-186673 Add CannotSetFieldAsTranslatableIfTranslationsIsDisabled test

victorhcunha committed:
LPS-186674 Add CannotAddTranslationToNonTranslatableField test

kristinonias committed:
LPS-184157 Fix SLAProcessingTimeFormatIsFollowingSystemLanguage test

kristinonias committed:
LPS-184157 Add tasks to SLAProcessingTimeFormatIsFollowingSystemLanguage test

victorhcunha committed:
LPS-186674 Add new macros to update object fields via api

kyle-miho committed:
LRQA-81055 Same as LPS-138761, solves a race condition that Data Guard detects as orphan data that should not be considered orphan data

kevenleone committed:
LPS-189271 Use SearchBuilder and Rename Extension from JS to TS

Brunno0liveira committed:
COMMERCE-11821 Add Macros addNewDiscountsRule and editDiscountRule

Brunno0liveira committed:
COMMERCE-11821 Add Test CanApplyDiscountWithTaxCategoryEnabled

kevenleone committed:
LPS-189271 Install SWR

brianikim committed:
COMMERCE-11494 Handle NPE for when CPDefinitionOptionRel or CPDefinitionOptionValueRel is null

Commit:
LPS-187665 Do not let delete custom object fields created before modifiable object definition publication via UI

Commit:
LPS-187665 Do not let delete custom object fields created before modifiable object definition publication via API

qz1010 committed:
LPS-156857 SF Remove unnecessary empty line

john-co committed:
LPS-188470 Add component name to feature flag test

john-co committed:
LPS-188470 Fix feature flags testcase so it can run properly on CI

john-co committed:
LPS-188470 Fix feature flags testcase so it can run properly on CI

ealonso committed:
LPS-189578 Adds external reference code when we are getting the fields of a collection

Commit:
LPS-188243 - Keep position fixed

Commit:
Revert "LPS-186839 Disable OrganizationSystemObjectDefinitionManager"

This reverts commit 3629e3cfd1ed92038fb6aa5e559fbc4231daa08b.

joelrocha68 committed:
LPS-189551 Add createOrganizationAPI macro

joelrocha68 committed:
LPS-189551 Add createRequestAPI macro

Commit:
COMMERCE-11251 rule should be disabled by default

ces-quanhuynh committed:
LPS-189561 Calendar event starts at 12am the same event will not  appear in the previoss day agenda

Commit:
LPS-186000 Add new icon field to Field

Commit:
LPS-188046 Using customizers to define custom icons as requested

sandrodw3 committed:
LPS-188046 Display icon when it's available

sandrodw3 committed:
LPS-189594 Add new URL and sort

joelrocha68 committed:
LPS-189551 Fix macro

Commit:
LPS-188046 Modifiying Field's creation with customized icon when necesary

sandrodw3 committed:
LPS-189594 Remove element only if it exists

sandrodw3 committed:
LPS-189594 Get default action error and use it when user doesn't fill any message

murilostodolni committed:
LPS-189612 Remove feature flag LPS-163716

ruben-pulido committed:
LPS-189594 Add getInfoItemActionErrorMessage to InfoItemActionExecutor interface

ruben-pulido committed:
LPS-189594 Implement getInfoItemActionErrorMessage in ObjectEntryInfoItemActionExecutor

murilostodolni committed:
LPS-189612 Remove feature flag LPS-163716 in the front end

DiegoHu97 committed:
LPS-183075 Remove default properties for master pages

DiegoHu97 committed:
LPS-183075 Remove default properties for layouts

Stephy6 committed:
LPS-176668 Use clay:alert in site-navigation-menu-web module

Commit:
LPS-187050 Fix test that doesn't see the Application Menu

evanilson-santana committed:
LPS-187709 Add CanChangeDataSource Test

BarbaraCabrera committed:
LPS-176671 Replace alert divs with clay:alert taglib in layout-admin-web

georgel-pop-lr committed:
LPS-189338 Remove script type 'text/javascript' as is no longer necessary.

Jeremy-Ch committed:
LPS-170175 Remove the unnecessary step

ruben-pulido committed:
LPS-189594 Add getInfoItemActionErrorMessageURL to config in ContentPageEditorDisplayContext

ruben-pulido committed:
LPS-189594 Create GetInfoItemActionErrorMessageMVCResourceCommand

Jeremy-Ch committed:
LPS-170175 Use fast navigation to add a document in site admin

Jeremy-Ch committed:
LPS-170175 Add folder by JSON

sandrodw3 committed:
LPS-189578 Add to useEffect dependencies

Jeremy-Ch committed:
LPS-170175 Add user to site by JSON

Jeremy-Ch committed:
LPS-170175 Remove the unnecessary navigation

Brunno0liveira committed:
COMMERCE-11821 Update Macro viewCheckoutOrderSummary and initializersCheckout

Commit:
LPS-186000 baseline

brianikim committed:
COMMERCE-11491 Handle NPE for when cpOption returns null

Stephy6 committed:
LPS-176675 Substitute alert with clay:alert and use clay:link instead if aui:button in layout-page-templates

Brunno0liveira committed:
COMMERCE-11494 Add Tests CanDeleteProductOptionWithGeneratedSkus and CanDeleteProductOptionFromOptionAdminPage

ambrinchaudhary committed:
LPS-189740 Only show allowed versions

sandrodw3 committed:
LPS-189578 Don't send classPK and ERC when they don't have a value

sandrodw3 committed:
LPS-189578 Display a warning when sending a param without value

ambrinchaudhary committed:
LPS-189740 Add the View More button

Stephy6 committed:
LPS-176675 Substitute alert with clay:alert and use clay:link in import process

Stephy6 committed:
LPS-176675 Substitute alert with clay:alert in layout-type-controller module

ruben-pulido committed:
LPS-189594 Refactor. Avoid extracting actionName from fieldId and use InfoItemFormProvider instead

ruben-pulido committed:
LPS-189594 Apply to executeInfoItemAction too

ruben-pulido committed:
LPS-189594 Adapt usages

ruben-pulido committed:
LPS-189594 SF. Reorder for better performance

victorhcunha committed:
LPS-188708 Fix widget button to new behavior in object test

jeffwu0724 committed:
LPS-189816 add "net.minidev" to shared-dependencies-minidev

jeffwu0724 committed:
LPS-189816 fix the typo in source-formatter. Because of this, auto sf does not work properly, and we have two versions of accessors-smart exist in the codebase, which are 2.4.2 and 2.4.9

jeffwu0724 committed:
LPS-189816 upgrade accessors-smart to 2.4.9, as we have two versions, we would like to keep the higher one to make it consistent. Also, 2.4.9 is the required version of json-smart 2.4.10

jeffwu0724 committed:
LPS-189816 auto SF

jeffwu0724 committed:
LPS-189816 change compileInclude to compileOnly, since we have net.minidev exported in shared-dependencies

jeffwu0724 committed:
LPS-189816 auto generate, by running "ant build-lib-versions"

BarbaraCabrera committed:
LPS-176671 Replace alert divs with clay:alert taglib but keeping the liferay-ui:message in layout-admin-web

sandrodw3 committed:
LPS-189594 Use empty string when text value is undefined

baharturk committed:
COMMERCE-11909 Add new JSON macros

baharturk committed:
COMMERCE-11909 Add a new path

baharturk committed:
COMMERCE-11909 Adapt tests according to the new functionality

ambrinchaudhary committed:
LPS-189740 link to the history page

ruben-pulido committed:
LPS-189594 Return Map with values for all available locales rather than a localized string

kevenleone committed:
LPS-189271 Replace System Feature by CX Feature Flag

kevenleone committed:
LPS-189271 Increase Height

ruben-pulido committed:
LPS-189594 Move getInfoItemActionErrorMessageMap to new Interface InfoItemActionDetailsProvider since it is not strictly related to action execution

ruben-pulido committed:
LPS-189594 Move getInfoItemActionErrorMessageMap to new class ObjectEntryInfoItemActionDetailsProvider

Brunno0liveira committed:
COMMERCE-11821 Add Macro goToTaxCategoriesAdmin

Brunno0liveira committed:
COMMERCE-11821 Add Macros editTaxCalculation and addFixedTaxRateTaxConfiguration

ruben-pulido committed:
LPS-189594 Adapt usages

lfbesada committed:
LPS-186000 Use the correct icon name

robertoDiaz committed:
LPS-189867 Do not use DownloadFileEntryItemSelectorReturnType for image field since is rendered as image tag

kevenleone committed:
LPS-189271 Adjust Translation and add spaces to text

Commit:
LPS-189603 | Add pagination in Team Members

kevenleone committed:
LPS-189271 Adjust Year Count

kevenleone committed:
LPS-189271 SF

CalumR23 committed:
POSHI-587 Generate CHANGELOG

ling-alan-huang committed:
LPS-179445 Auto SF

michaelhashimoto committed:
LRCI-3616 Add poshi queries to test relevant hotfix changes

michaelhashimoto committed:
LRCI-3616 Enable testing against hotfix changes

Brunno0liveira committed:
COMMERCE-11494 Add Path SKU_STATUS_PRODUCT_ADMIN_PAGE

danielCamara02 committed:
LPS-185340 Add ConfigureCollectionDisplayThatReturnsBlueprintAssets test

Commit:
LPS-188470 Update FeatureFlag.testcase

Commit:
LPS-189934 Fix in image redering 2

Commit:
LPS-189935 Add accountEntryRestrictedFieldName

drewbrokke committed:
POSHI-556 poshi-vscode: updates repo and readme fields

- Fixes the repository link to include a .git suffix
- Adds the relative directory (ignored by marketplace but still correct)
- Adds a relative link to the README.md file

drewbrokke committed:
POSHI-556 .github: do not use skip duplicate option for now until the vsce bug is fixed

brittneyq committed:
LRCI-3661 Add test result to failures if the batch is failing and there is only 1 result

brittneyq committed:
LRCI-3661 Add passed test result if there are no other test results

holatuwol committed:
LPS-189946 Introduce a new API that returns a document directly from JournalConverter

holatuwol committed:
LPS-189946 Leverage new API to skip the document to string to document conversion

holatuwol committed:
LPS-189946 Baseline

Brunno0liveira committed:
COMMERCE-11821 Add Macro goToDiscountAdmin

lfbesada committed:
LPS-186000 SF Sort

Commit:
LPS-176671 Modify macro to adapt new markup

liulukiwi committed:
LPS-176675 Update alert message in macro

Commit:
LPS-189951 Test fix for NestedFieldsContainerRequestFilterTest.testFilter()

Commit:
LPS-187911 Sort

Commit:
LPS-187911 Refactor case

Commit:
LPS-187911 Remove redundant steps and Sort

Commit:
LPS-187911 Remove redundant steps

Commit:
LPS-187911 Add step to check the warning message on modal

Commit:
LPS-187911 Remove setup virtual instance and tear down block

Jeremy-Ch committed:
LPS-170175 Rename the tests and change the refactor status

JessieZhao0714 committed:
LRQA-81388 Update the task description

Jeremy-Ch committed:
LPS-170175 Add macro for adding multiple documents by JSON

p2kmgcl committed:
LPS-189943 Update popover language key

lfbesada committed:
LPS-189578 Fix basic-document subtype label

lfbesada committed:
LPS-189578 Remove unused usagesCount property

lfbesada committed:
LPS-189578 Adapt JS unit tests

lfbesada committed:
LPS-189578 Delete unused service method

lfbesada committed:
LPS-189578 BuildService

lfbesada committed:
LPS-189578 Delete related integration tests

robertoDiaz committed:
LPS-189976 In a clustered environment the empty attribute could be an Object[] instead of a String[] and could produce a Cast error

robertoDiaz committed:
LPS-189976 Also for categories

victorg1991 committed:
LPS-189843 Check update permission for showing convert actions

ambrinchaudhary committed:
LPS-189740 Improve conditionals

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-189985 Enable AssetCategorySegmentsFieldCustomizer by default

victorg1991 committed:
LPS-189843 Check update permission for showing convert actions

victorg1991 committed:
LPS-189993 Change icon color

BarbaraCabrera committed:
LPS-176668 Adjust some clay alerts to follow the lexicon patterns

victorg1991 committed:
LPS-189996 Isolate form container select

joelrocha68 committed:
LPS-188879 Add goToCreateRequest macro

joelrocha68 committed:
LPS-188879 Add VerifyStatusAfterCreatRequest test

gislayne-vitorino committed:
LPS-187409 Add CTAware to support site headless service

drewbrokke committed:
POSHI-556 poshi-vscode: renames README.markdown to README.md

drewbrokke committed:
POSHI-556 poshi-vscode: adds baseImagesUrl property to vsce configuration

Kvale1733 committed:
Revert "LRQA-80656 Set portalURL first before getting companyId so macro can be used for any specific url"

This reverts commit fce3d306a5edeb31712ac3303b2abb3545ee40a5.

drewbrokke committed:
POSHI-556 poshi-vscode: updates extension display name

Kvale1733 committed:
LRQA-81392 Sign in with UI instead

drewbrokke committed:
POSHI-556 poshi-vscode: bumps version to 0.3.6

drewbrokke committed:
POSHI-556 poshi-vscode: adds version release notes

drewbrokke committed:
POSHI-556 poshi-vscode: regen package-lock.json

joelrocha68 committed:
LPS-166354 Add CanCreateRequestAfterCreateOrganization test

joelrocha68 committed:
LPS-166354 Add CanSeeRedirectionToOrganizationPageAfterCreateOrganization test

joelrocha68 committed:
LPS-166354 Add CanseeSuccessMessageAfterCreateOrganization test

drewbrokke committed:
POSHI-556 poshi-vscode: makes the other version range presentation consistent

drewbrokke committed:
POSHI-556 poshi-vscode: adds icon

Commit:
LPS-189518 Fix Calendar tests

joelrocha68 committed:
LPS-166354 Add viewSuccessMessage macro

joshchong committed:
LPS-190046 Set single-node discovery as default

Commit:
LPS-189550 Do not support update object action

joelrocha68 committed:
LPS-166354 Fix test

JerryAdrianoo committed:
LPS-190045 Remove ignore property

ryanschuhler committed:
ISSITES-6034 Update post function

CalumR23 committed:
POSHI-587 Update poshi core version

CalumR23 committed:
POSHI-587 Update default poshi runner version

xbrianlee committed:
LPS-187409 Navigate to next page if the change category is not present

xbrianlee committed:
LPS-187409 Update change order

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-190063 Run cases in isolation to avoid being affected by previous cases

Commit:
LPS-187238 Use jsonws macros to instead of UI macros

Commit:
LPS-187238 Remove unnecessary steps

Commit:
LPS-187238 Add macro to import structure

Commit:
LPS-187238 Modify cases to import structure instead of create

Commit:
LPS-187238 Add json file for web content structure with all fields

Commit:
LPS-187238 Modify macro to support Boolean field

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-187238 Remove set up virtual instance and tear down block

Commit:
LRAC-13850 Fix takeTokenFromAnalyticsCloud macro after the Faro migration

Commit:
LPS-190065 Update parameters

Commit:
LPS-190066 Add function to type keys

Commit:
LPS-190066 Add macro to change pagination via keyboard

Commit:
LPS-190066 Modify macro to change pagination via keyboard

Commit:
LPS-190066 Fix typo

Commit:
LPS-190066 Disable ci retries

Commit:
LPS-190074 Modify case to adapt new behavior

Commit:
LPS-190076 Modify macro to adapt new markup

JoyceWang08211 committed:
LRQA-81405 Update case

lfbesada committed:
LPS-189933 Adds FF annotation

brianchandotcom committed:
Revert "LPS-186063 Can we do it in this order?"

This reverts commit ebe302087c33a56697facfe0adeb2c13a9ea1529.

brianchandotcom committed:
LPS-186063 reverted, and republished

brianchandotcom committed:
LPS-186063 refresh other workspaces

brianchandotcom committed:
POSHI-587 prep next

brianchandotcom committed:
POSHI-587 prep next

lfbesada committed:
LPS-190082 Uses the correct tag definition

lfbesada committed:
LPS-190082 Adds required dependencies

liferay-translation committed:
LPS-77699 Update Translations

claraizquierdo committed:
LPS-190088 Change LPS of the Feature Flag for story LPS-186155

adolfopa committed:
LPS-189740 SF

Commit:
LPS-187238 Add json file for web content structure with contact information fields

Commit:
LPS-187238 Simplify cases

Commit:
LPS-187238 Fix typo

Commit:
LPS-190075 Modify case to adapt new message

sandrodw3 committed:
LPS-185441 Change alert copy

sandrodw3 committed:
LPS-185441 buildLang

brianchandotcom committed:
LRCI-3661 SF

brianchandotcom committed:
LRCI-3661 prep next

sandrodw3 committed:
LPS-176668 Fix spacing

Commit:
LPS-189492 Fix KaleoformsUsecase tests

ealonso committed:
LPS-190141 Uses correct permissions class

sandrodw3 committed:
LPS-190159 Modify key as it doesn't support parenthesis

sandrodw3 committed:
LPS-190159 buildLang

Commit:
COMMERCE-11938 change type of commerce notification columns

Commit:
COMMERCE-11938 gw buildService

Commit:
COMMERCE-11938 gw baseline

pyoo47 committed:
LRCI-3782 Override equals()

Commit:
LPS-190171 Add step to wait for page load

Commit:
LPS-190171 Use UI macros to add collections

pyoo47 committed:
LRCI-3782 Push cached branch to nodes that don't have it

Commit:
LPS-190177 Fix typo

pyoo47 committed:
LRCI-3782 Override hashCode()

Commit:
LPS-190189 Modify macros to make them reusable

Commit:
LPS-188566 Add macro to edit text of input field

Commit:
LPS-188566 Add case to configure API key

Commit:
LPS-188566 Add locator for loading animation on modal

Commit:
LPS-188566 Refactor macro

Commit:
LPS-188566 Add case to generate content

Commit:
LPS-188566 Add macro to add generated content

Commit:
LPS-188566 Add case to view generated content appended to existing content

Commit:
LPS-188566 Add case to view focus on modal when AI Creator modal is open

Commit:
LPS-188566 Add case to view accessibility of AI Creator modal

Commit:
LPS-190115 Update coverage info

Commit:
LPS-190115 Add case to view error message when provide a wrong API key

Commit:
LPS-190115 Add case to view error message when OpenAI server errors happen on generating content

Commit:
LPS-190115 Add case to view error message when generic errors happen on configuring API key

brianchandotcom committed:
LPS-189976 SF

Commit:
LPS-190115 Add case to view error message when generic errors happen on generating content

brianchandotcom committed:
LRCI-3782 wordsmith

brianchandotcom committed:
LRCI-3782 SF

brianchandotcom committed:
LRCI-3782 prep next

liferay-translation committed:
LPS-77699 Update Translations

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorcdp commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/c641fd699669191c21f5c20429da55fc6a1e5490">c641fd699669191c21f5c20429da55fc6a1e5490</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorcdp/liferay-portal/tree/COMMERCE-11938">COMMERCE-11938</a><br/>Branch GIT ID: <a href="https://github.com/victorcdp/liferay-portal/commit/092dff5890e154031ae35f9e03d29d40cde67869">092dff5890e154031ae35f9e03d29d40cde67869</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-source-format/7563/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/7563/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-source-format/7563/">test-portal-source-format#7563</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-source-format/builds/7563/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/brianikim/liferay-portal/pull/99">brianikim#99</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2378048008">[master] ci:test:sf - victorcdp &gt; brianikim - PR#99 - 2023-07-11[08:20:07]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/28358/">publish-testray-report#28358</a><br/></div></div>

brianikim commented:
Seems fine. You can forward.
